sonar-php-plugin/src/test/java/org/sonar/plugins/php/PhpPluginTest.java

```diff
@@ -51,4 +55,28 @@ public void test_sonarlint() {
     assertThat(context.getExtensions()).hasSize(9);
   }
 
+  @Test
+  public void should_contain_REPORT_PATHS_after_6_2() throws Exception {
+    final Plugin.Context context6_2 = qubeContext(Version.create(6, 2));
```

Comment:
Yes, and I also added qube to IntelliJ's dictionary, so that it does not complain :D